README.rst

```diff
@@ -38,7 +32,14 @@ Additional mailbox functionality.
 Features
 --------
 
-* TODO
+* ``filelock.Filelock`` offers a file locking mechanism that has context-manager support so
+    you can use it in a with statement.
+
+* ``smaildir.Maildir`` offers support for maildir mailbox. It uses scandir() instead of listdir(). It also provides a new iter_folders() method that works like list_folders(), put provides a generator instead of returning a list.
+
+* ``se_mailbox.QuotaMixin`` implements the Maildir++ quota size system, as described here: http://www.inter7.com/courierimap/README.maildirquota.html. Quotas are not enforced - this would be good to add, but since we do a non-standard operation (dropping old data instead of failing to add) there's not much point.
```

Comment:
I think the "since we do a non-standard operation" part of this should probably not be there on a public repo.